conda/gateways/connection/session.py

```diff
@@ -96,7 +96,9 @@ class CondaHttpAuth(AuthBase):
     # TODO: make this class thread-safe by adding some of the requests.auth.HTTPDigestAuth() code
 
     def __call__(self, request):
-        request.url = CondaHttpAuth.add_binstar_token(request.url)
+        request.url, token = CondaHttpAuth.add_binstar_token(request.url)
+        if token is not None:
+            request.headers["Authorization"] = "Bearer {0}".format(token)
```

Comment:
Is there a reason for having the token prepended with "Bearer"? It seems not super necessary to me

Reply:
https://jwt.io/introduction/ this article explains that pattern pretty well